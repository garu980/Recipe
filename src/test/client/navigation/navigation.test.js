import { shallow } from 'enzyme';
import * as React from 'react';
import Navigation from '../../../client/presentation/navigation/navigation';
import * as Redux from 'react-redux';

describe('Navigation rendering', () => {
  it('renders the component', () => {
    jest
      .spyOn(Redux, 'useSelector')
      .mockImplementation((state) => state.language)
      .mockReturnValue({ language: 'en' });
    const navigation = shallow(<Navigation />);
    expect(navigation).toMatchSnapshot();
  });
});
