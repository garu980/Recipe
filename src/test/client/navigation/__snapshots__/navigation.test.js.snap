// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation rendering renders the component 1`] = `ShallowWrapper {}`;
